src/components/HTMLEngineProvider/HTMLRenderers/AnchorRenderer.tsx

```diff
@@ -58,18 +62,16 @@ function AnchorRenderer(props) {
             // eslint-disable-next-line react/jsx-props-no-multi-spaces
             target={htmlAttribs.target || '_blank'}
             rel={htmlAttribs.rel || 'noopener noreferrer'}
-            style={{...props.style, ...parentStyle, ...styles.textUnderlinePositionUnder, ...styles.textDecorationSkipInkNone}}
-            key={props.key}
-            displayName={displayName}
+            style={[{...parentStyle, ...styles.textUnderlinePositionUnder, ...styles.textDecorationSkipInkNone}, style]}
```

Comment:
```suggestion
            style={[parentStyle, styles.textUnderlinePositionUnder, styles.textDecorationSkipInkNone, style]}
```
I think it can be simplified